components/prize_pool/commons/prize_pool.lua

```diff
@@ -892,7 +893,9 @@ function Placement:_getLpdbData()
 			lastscore = (opponent.additionalData.LASTVSSCORE or {}).score,
 			lastvsscore = (opponent.additionalData.LASTVSSCORE or {}).vsscore,
 			groupscore = opponent.additionalData.GROUPSCORE,
-			extradata = {}
+			extradata = {
+				prizepoints = tostring(pointsReward or ''),
```

Comment:
why the tostring?
wouldn't numbers be better?

Reply:
To match storage behavior in current live versions (and it's will be set as a string later in the TeamCard storage anyway).
With the fields to be added, there it will likely be a number.

Reply:
alright